### ISSUE TITLE
Map based job browsing

### ISSUE THREAD

**Issue by jdries** [Open-EO/openeo-web-editor#302]
Batch jobs are now visualized in a list, but sometimes it's easier to find a job by it's location.
Sometimes failing jobs are also related to specific locations, and a map view can reveal this more easily.

A feature like this would also help in tracking large area processing.

**Comment by m-mohr** [comment]
@jdries Yeah, it needs good metadata though. Do you have an example that I can look at without running such a large job just for the metadata?

**Comment by m-mohr** [comment]
Wait a second, this is meant not for a single job but for all of them, right?
Then I'd need to request all results, which seems pretty unfeasible as it may end up in 100s of requests (depending on the number of jobs).
